docs/guides/01-css-elements.md

```diff
@@ -0,0 +1,127 @@
+# CSS Elements
+
+## Installation
+
+### From NPM
+  #### Install @carto/airship-style package from npm
+  ```
+  npm i @carto/airship-style --save
+  ```
+
+  #### Importing styles in SASS, PostCSS
+  ```css
+  // Import whole bundle
+  @import '~@carto/airship-style';
+
+  // Import SASS element styles
+  @import '~@carto/airship-style/src/{path}';
+
+  // Import CSS element styles
+  @import '~@carto/airship-style/dist/{path}';
+  ```
+
+  #### Importing styles in JavaScript
+  ```javascript
+  // Import whole bundle
+  import '@carto/airship-style';
+
+  // Import SASS element styles
+  import '@carto/airship-style/src/{path}';
+
+  // Import CSS element styles
+  import '@carto/airship-style/dist/{path}';
+  ```
+
+### From CDN
+Copy-paste the stylesheet `<link>` tag into your `<head>` tag before other stylesheets to load Airship CSS.
+```html
+<link rel="stylesheet" href="https://libs.cartocdn.com/airship-styles/<VERSION>/airship.css" crossorigin="anonymous" integrity="">
+```
+
+## Usage
+
+After importing the package in your current application, you only need to set some CSS classes to your current HTML, or use a components' example HTML content to test it out.
+
+For example, you can show a button with this HTML scaffolding:
+```html
+<button class="as-btn as-btn--primary">
+  I'm a regular button
+</button>
+```
+
+Please check [styles documentation]() (missing link) out and see how to use Airship components.
+
+## Framework integration
+To integrate Airship into your framework you only need to import the styles in the way you prefer from the ones stated above.
+
+## Browser Compatibility
```

Comment:
Usually this section is at the end of the doc

Reply:
We can put it at the end of the doc!